sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntimeConfig.java

```diff
@@ -36,6 +36,10 @@ public class ActorRuntimeConfig {
 
   private volatile Integer remindersStoragePartitions;
 
+  private volatile ActorReentrancyConfig actorReentrancyConfig;

```

Comment:
Please, remove reentrancy since it does not work with Java SDK yet. We might not even want that anymore in runtime.

Reply:
done